### PR TITLE
fix: typeface page styles and include Carbon Slider

### LINKED ISSUE #84: Typeface page style fixes
- [x] Replace lead video with new one from Buck
- [x] Sub-families section type should be bigger. Also the two columns of type should sit on page column grid.
**expected**
image

- [x] Type-tester sample text should be bigger and “Light”
- [x] Use default Carbon styles for slider
**expected**
image
image

### LINKED ISSUE #130: Typeace - subfamilies - mobile layout bug
Should be one column at mobile breakpoint

image

https://design-language-website.netlify.com/elements/typeface#sub-families

### LINKED ISSUE #131: Typeface - Type tester - text should be light as default
The default should be Plex Light

image

### PR DESCRIPTION
**Pull request by emyarod**
Closes #84
Closes #130 
Closes #131 

This PR addresses several style issues on the Typeface page

#### Changelog

**New**

- responsive font size (values lifted from current IDL site)

**Changed**

- Replace `<InputRange>` with Carbon `<Slider>`
- typeface page lead video ID

### CODE REVIEW

**Comment by alisonjoseph** [comment]
I think we can get rid of the `padding: 0 1rem` on `.bx--type-tester-menu .bx--input-range-wrapper` since it is already built in to the Carbon slider.

**Comment by emyarod** [comment]
removed it @alisonjoseph , now just waiting for the new lead video

**Comment by alisonjoseph** [comment]
new vimeo ID for the top video is 337384869

**Comment by emyarod** [comment]
got it, this PR should be ready for review now

**Comment by emyarod** [comment]
just wanted to get design review, didn't mean to request another review @alisonjoseph !

**Comment by jeanservaas** [comment]
Hey @alisonjoseph real quick just noticed that on IDL this is a big as the mono ever gets

image


In the gatsby component it keeps growing until the first col runs into the second column and it actually even breaks (I'm testing in Firefox). 

image


Slider seems to be good, that's the only issue I noticed not quite sure what the solve it, but those type sizes don't seem to be going as big (for the first component) in IDL.

**Comment by alisonjoseph** [comment]
@emyarod see @jeanservaas comment above. Looks like at the max breakpoint we need to set `font-size: 3.75rem;`  `@include carbon--breakpoint('max')`

**Comment by emyarod** [comment]
@alisonjoseph just to confirm, so we don't want it to be 3.75rem at the max breakpoint?

**Comment by alisonjoseph** [comment]
Yea, sorry, we should use the type token for expressive heading 05

File: src/components/TypeFaceSubFamilies/_typeface-sub-families.scss [hunk 1]
```diff
@@ -70,6 +70,23 @@
...
+  @include carbon--breakpoint('lg') {
+    font-size: calc(2.625rem + 0.375 * (100vw - 66rem) / 16);
+  }
+
+  @include carbon--breakpoint('xlg') {
+    font-size: calc(3rem + 0.75 * (100vw - 82rem) / 17);
+  }
+
+  @include carbon--breakpoint('max') {
+    @include carbon--type-style('expressive-heading-05');
```
Comment (alisonjoseph) [inline src/components/TypeFaceSubFamilies/_typeface-sub-families.scss]:
```suggestion
    @include carbon--type-style('expressive-heading-05', true);
```

We need this to get the correct responsive type size I believe.

**Comment by jeanservaas** [comment]
Width issue looks good now but I picked up one other thing.

On our G10 background, these tab hovers need to be e5e5e5, right now they are gray 10 also and they are totally disappearing.

image

**Comment by emyarod** [comment]
should be resolved now @jeanservaas

**Comment by jeanservaas** [comment]
Hey @emyarod 

Hover looks good (not sure what's going on with that browser focus in Chrome...)

But still getting a much larger max size typeface on the migrated IDL vs. the original

**preview link:**

image



**original IDL:**

image

File: src/components/TypeFaceSubFamilies/_typeface-sub-families.scss [hunk 2]
```diff
@@ -70,6 +70,21 @@
...
+    flex-direction: row;
+    font-size: calc(2.25rem + 0.375 * (100vw - 42rem) / 24);
+  }
+
+  @include carbon--breakpoint('lg') {
+    font-size: calc(2.625rem + 0.375 * (100vw - 66rem) / 16);
+  }
+
+  @include carbon--breakpoint('xlg') {
+    font-size: calc(3rem + 0.75 * (100vw - 82rem) / 17);
```
Comment (alisonjoseph) [inline src/components/TypeFaceSubFamilies/_typeface-sub-families.scss]: I think this should be 
`@include carbon--type-style('expressive-heading-05', true);`

??

Comment (emyarod) [reply src/components/TypeFaceSubFamilies/_typeface-sub-families.scss]: I'm not familiar with that mixin but it looks like it includes all of the media query rules already? (see latest commit)

**Comment by emyarod** [comment]
should be resolved now @jeanservaas

**Pull request merged**